workers/generic-worker/errorreport/errorreport.go

```diff
@@ -0,0 +1,28 @@
+package errorreport
+
+import (
+	"github.com/taskcluster/taskcluster/v30/internal/workerproto"
+)
+
+func Send(proto *workerproto.Protocol, err error, debugInfo map[string]string) {
```

Comment:
Is it worth returning an `error` to indicate if the message could be sent or not? Even if it isn't evaluated by the receiver.

Reply:
Oh wait - [`(workerproto.Transport).Send`](https://godoc.org/github.com/taskcluster/taskcluster/internal/workerproto#Transport) doesn't return an `error`?!?

Reply:
@djmitche Why does `PipeTransport` [silently fail](https://github.com/taskcluster/taskcluster/blob/ae8e898abac9821c48b2bf04f5f56561bdd7b690/internal/workerproto/pipe.go#L46-L49) without informing the caller? How can the worker know that its messages are processed?

Reply:
Having reflected on this a little more, I realised the `Send` method is essentially just writing to standard out, so we probably don't need to care too much about an error there, in the same way that we typically don't check the error returned by `fmt.Println(string)`. Note `fmt.Println` does actually return an error, but we just typically ignore it.